NEWS.md

```diff
@@ -169,52 +199,56 @@ More details can be found here: [Topology-aware AllReduce](https://cwiki.apache.
   * Test classification with LSTMs - 12263
 
 #### Scala
-* explain the details for Scala Experimental (#12348)
+* Explain the details for Scala Experimental (#12348)
 * MXNET-873 - Bring Clojure Package Inline with New DataDesc and Layout in Scala Package (#12387)
 * [MXNET-716] Adding Scala Inference Benchmarks (#12721)
 * [MXNET-716][MIRROR #12723] Scala Benchmark Extension pack (#12758)
 * NativeResource Management in Scala (#12647)
-* Ignore generated scala files. (#12928)
-* use ResourceScope in Model/Trainer/FeedForward.scala (#12882)
+* Ignore generated Scala files (#12928)
+* Use ResourceScope in Model/Trainer/FeedForward.scala (#12882)
 * [MXNET-1180] Scala Image API (#12995) 
-* ONNX export: Scalar, Reshape - Set appropriate tensor type (#13067)
-* Port of scala Image API to clojure (#13107) 
-* update log4j version of Scala package (#13131)
-* review require() usages to add meaningful messages. (#12570)
+* Port of Scala Image API to Clojure (#13107) 
+* Update log4j version of Scala package (#13131)
+* Review require() usages to add meaningful messages (#12570)
+* Fix Scala readme (#13082)
 
 #### Clojure
-* Introduction to Clojure-MXNet video link. (#12754)
+* Introduction to Clojure-MXNet video link (#12754)
 * Improve the Clojure Package README to Make it Easier to Get Started (#12881)
 
 #### Perl
 * [MXNET-1026] [Perl] Sync with recent changes in Python's API (#12739)
 
 #### Julia
-* import Julia binding
+* Import Julia binding
```

Comment:
Can we get a PR link here? 
https://github.com/apache/incubator-mxnet/pull/10149
I was curious to know how big this is... and it is pretty big so we should have some links going somewhere... Also something to a README about how to use it. Maybe this? http://dmlc.ml/MXNet.jl/latest/user-guide/install/

Reply:
added